components/content-service/cmd/run.go

```diff
@@ -33,6 +35,12 @@ var runCmd = &cobra.Command{
 		cfg := getConfig()
 		reg := prometheus.NewRegistry()
 
+		grpclog.SetLoggerV2(grpclog.NewLoggerV2(
+			log.WithField("component", "grpc").WriterLevel(logrus.InfoLevel),
+			log.WithField("component", "grpc").WriterLevel(logrus.WarnLevel),
+			log.WithField("component", "grpc").WriterLevel(logrus.ErrorLevel),
+		))
```

Comment:
I reckon it would make sense to move this to `common-go/grpc`, no?

Reply:
done